sw/device/tests/penetrationtests/firmware/lib/pentest_lib.c

```diff
@@ -355,8 +568,25 @@ status_t pentest_configure_cpu(
   if (disable_dummy_instr) {
     cpuctrl_csr = bitfield_field32_write(
         cpuctrl_csr, (bitfield_field32_t){.mask = 0x1, .index = 2}, 0);
+  } else {
+    cpuctrl_csr = bitfield_field32_write(
+        cpuctrl_csr, (bitfield_field32_t){.mask = 0x1, .index = 2}, 1);
+    // 0 Dummy instruction every 0 - 4 real instructions
+    // 1 Dummy instruction every 0 - 8 real instructions
+    // 3 Dummy instruction every 0 - 16 real instructions
+    // 7 Dummy instruction every 0 - 32 real instructions
+    cpuctrl_csr = bitfield_field32_write(
+        cpuctrl_csr, (bitfield_field32_t){.mask = 0x7, .index = 3},
+        dummy_instr_count);
+    CSR_WRITE(CSR_REG_CPUCTRL, cpuctrl_csr);
+    uint32_t check_csr;
+    CSR_READ(CSR_REG_CPUCTRL, &check_csr);
+    if (check_csr == cpuctrl_csr) {
+      LOG_INFO("CPUCTRL register correctly set");
+    } else {
+      LOG_INFO("Something in CPUCTRL was locked");
```

Comment:
Would it make sense to abort the test to avoid that this print is missed by the evaluator and thinks that everything is OK?
```suggestion
      LOG_INFO("Something in CPUCTRL was locked");
      return ABORTED();
```